php/php-pcov/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               php 1.1
+
+name                    php-pcov
+version                 1.0.11
+categories-append       devel
+maintainers             nomaintainer
+license                 Apache-2
```

Comment:
```suggestion
license                 PHP-3.01
```

Reply:
Done, silly me, I didn't check that. Thanks!